GitUI/UserControls/FileStatusList.cs

```diff
@@ -970,7 +973,29 @@ private void UpdateFileStatusListView(bool updateCausedByFilter = false)
                     FileStatusListView.Groups.Add(group);
                 }
 
-                foreach (var item in i.statuses)
+                if (isFirstGroup && i.statuses.Count < 1)
+                {
+                    itemStatuses = noItemStatuses;
+                }
+                else
+                {
+                    itemStatuses = i.statuses;
+                }
+
+                if (GitItemStatusesWithDescription.Count > 1)
```

Comment:
Why not `> 0`?
```suggestion
                if (GitItemStatusesWithDescription.Count > 0)
```

Reply:
(Not confirmed this, but the ListGroup is Collapsible by default so this is not needed. But the code would be simpler if the compare and set state is removed.)

Reply:
Yes, can be 0. Thank you